news/news-4.0.0-beta-1.md

```diff
@@ -26,39 +25,28 @@ Some of the highlights include:
-- Transmission now uses Sonarcloud, Coverity, LGTM, and clang-tidy static analysis on new code. Hundreds of code warnings have been fixed compared to Transmission 3.00.
+- The Web client has been rewritten in modern JavaScript and no longer uses jQuery. The entire gzipped bundle is now 68K.
+- The unit tests have been expanded and ported to [Google Test](https://github.com/google/googletest). Clang sanitizer builds are run during CI.
+- The core library is now fuzz tested.
+- Transmission now uses Sonarcloud, Coverity, LGTM, and clang-tidy static analysis on new code. Several hundred code warnings have been fixed compared to Transmission 3.00.
 
 ### New Features
 
-- Transmission 4.0.0 now supports .torrent files that were created with the [BitTorrent v2 spec](http://bittorrent.org/beps/bep_0052.html). 4.0.0 users will be able to download and seed torrents created by both BitTorrent v1 and v2. Support for _creating_ v2 torrents is slated for the next major release of Transmission.
-- Support setting "default" trackers that can be used for announcing all public torrents
-- Newly-added seeds can start immediately and verify pieces as needed, instead of requiring a full verify before seeding can begin ([#2626](https://github.com/transmission/transmission/pull/2626))
-- Added `transmission-create` option to omit potentially-identifying information, such as User-Agent and date created, when creating torrents. ([#3452](https://github.com/transmission/transmission/pull/3452))
-- Dozens of other new features; too many to list here!
-
-### Web Client
-
-The web client has been given a major overhaul! User-visible highlights include:
-* Mobile is now fully supported.
-* Added fullscreen support on mobile.
-* Better support for dark mode.
-* Added mime icons to the torrent list.
-* Improved theme consistency across the app.
-
-Maintainer highlights include:
-* Updated code to use modern JavaScript.
-* No longer use jQuery UI.
-* No longer use jQuery.
-* Use Webpack to bundle the Javascript, CSS, and assets together -- the entire bundle size is now 68K gzipped.
-* Added eslint / prettier / stylelint tooling.
-* Uses torrent-get's 'table' mode for more efficient RPC calls.
+- Support for using [BitTorrent v2](http://bittorrent.org/beps/bep_0052.html) torrents and [hybrid](http://bittorrent.org/beps/bep_0052.html#upgrade-path) torrents. (Support for _creating_ v2 and hybrid torrents is slated for an upcoming release.)
+- Users can now set "default" trackers that can be used to announce all public torrents.
+- Newly-added seeds can start immediately and verify pieces on demand, instead of needing a full verify before seeding can begin. ([#2626](https://github.com/transmission/transmission/pull/2626))
+- There now an option to omit potentially-identifying information (e.g. User-Agent and date created) when creating new torrents. ([#3452](https://github.com/transmission/transmission/pull/3452))
```

Comment:
"There now" -> "There is now"

Reply:
Fixed in 32d02e5f9bd622fe9cfab8b7828390a496e5aa37